resources/openapi/yaml/contractnegotiation.yaml

```diff
@@ -0,0 +1,122 @@
+openapi: 3.0.1
```

Comment:
there are two yaml files: `contract-negotiation.yaml` and `contractnegotiaion.yaml`, we only need one.

Reply:
Good catch - is a renaming leftover. Will fix it